packages/playwright/src/runner/testServer.ts

```diff
@@ -308,6 +308,7 @@ class TestServerDispatcher implements TestServerInterface {
       reporter: params.reporters ? params.reporters.map(r => [r]) : undefined,
       use: {
         trace: params.trace === 'on' ? { mode: 'on', sources: false, _live: true } : (params.trace === 'off' ? 'off' : undefined),
+        video: 'off',
```

Comment:
```suggestion
        video: params.reuseContext ? 'off' : undefined,
```

Reply:
As agreed offline, video is primary and reuseContext is best effort here.